modules/formulize/include/formulizeConfigSync.php

```diff
@@ -729,8 +738,12 @@ private function exportElementsForForm(int $formId): array
 	 * @param array $elementRow Raw element data from database
 	 * @return array Prepared element data for export
 	 */
-	private function prepareElementForExport(array $elementRow, array $dataTypes): array
+	private function prepareElementForExport(array $elementRow): array
 	{
+
+		$elementHandler = $this->elementHandler->get($elementRow['ele_handle']);
```

Comment:
naming issue... I would strongly suggest we call this $elementObject not $elementHandler

Reply:
Committed now. Kinda wild you can invoke "Codespace" in Github and it's VSCode in a browser.